src/Microsoft.AspNetCore.Blazor.Templates/content/BlazorServerSide-CSharp/BlazorServerSide-CSharp.Client/Program.cs

```diff
@@ -0,0 +1,16 @@
+﻿using Microsoft.AspNetCore.Blazor.Hosting;
+
+namespace BlazorServerSide_CSharp.Client
```

Comment:
I think we agreed that .Client was confusing when running Blazor on the service. Instead we had proposed .App and .Server.

Reply:
Oh yes, well remembered!